src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/RoleAssignmentFilter.java

```diff
@@ -241,9 +251,9 @@ private static Predicate getEmptyOrNullAuthorizationsPredicate(
     private static Predicate mapSecurityClassification(
         Root<TaskResource> root,
         CriteriaBuilder builder,
-        RoleAssignment roleAssignment) {
-        final Classification classification = roleAssignment.getClassification();
-        if (classification != null) {
+        RoleAssignmentForSearch roleAssignment) {
+        if (roleAssignment.getClassification() != null) {
```

Comment:
New line following re-engineering for performance. Can we add unit test to cover this line?